src/test/kotlin/racingcar/domain/CarsTest.kt

```diff
@@ -2,57 +2,55 @@ package racingcar.domain
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.assertDoesNotThrow
-import org.junit.jupiter.api.assertThrows
+import racingcar.domain.movestrategy.MoveStrategy
 
 internal class CarsTest {
 
-    @Test
-    fun `elements numbers 사이즈가 다를 경우`() {
-        val car1 = Car(Name("test"))
-        val car2 = Car(Name("test"))
-        assertThrows<IllegalArgumentException> { Cars(listOf(car1, car2)).move(listOf()) }
+    private val moveStrategy = object : MoveStrategy {
+        override fun movable() = true
     }
 
-    @Test
-    fun `elements numbers 사이즈가 같을 경우`() {
-        val car1 = Car(Name("test"))
-        val car2 = Car(Name("test"))
-        assertDoesNotThrow { Cars(listOf(car1, car2)).move(listOf(Number(), Number())) }
+    private val notMoveStrategy = object : MoveStrategy {
+        override fun movable() = false
     }
 
     @Test
     fun `움직이지 않는 경우`() {
         val car1 = Car(Name("test"))
         val car2 = Car(Name("test"))
         val cars = Cars(listOf(car1, car2))
```

Comment:
재사용하지 않는 지역 변수의 수는 최소화하는 것이 좋아요.

```suggestion
        val cars = Cars(listOf(Car(Name("test")), Car(Name("test"))))
```

Reply:
이 경우는 제이슨 말이 맞지만 경우에 따라서 저는 지역변수로 가독성을 높일 수 있다고 생각해요 . 제이슨 생각은 어떤가요?

Reply:
커다란 표현을 쪼개는 가장 쉬운 방법은 작은 하위 표현을 담을 추가 변수를 만드는 것이며 하위 표현의 의미를 설명하므로 설명 변수(explaining variable)라고도 합니다. 협업하는 팀원이 쉽게 읽을 수 있는 코드가 좋겠죠?